README.md

```diff
@@ -20,7 +21,7 @@ JSON Patch plays well with the HTTP PATCH verb (method) and REST style programmi
 Mark Nottingham has a [nice blog]( http://www.mnot.net/blog/2012/09/05/patch) about it.
 
 ## Footprint
-1.22 KB minified and gzipped (3 KB minified)
+4 KB minified and gzipped (12 KB minified)
```

Comment:
It's not 4 times TBH. This 1.22 was for `json-patch.js` only. It's kinda cheating.